meson.build

```diff
@@ -1636,7 +1636,12 @@ if get_option('efi')
 else
         have = false
 endif
+
+subdir('src/fundamental')
+subdir('src/boot/efi')
+
 conf.set10('ENABLE_EFI', have)
```

Comment:
This uses `have`, which should refer to the check that was done above. But when we exectue the subdirs first, `have` might be reassigned. So either the assignement of `ENABLE_EFI` should be moved up, or this needs to be rearranged somehow.